app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -508,6 +507,10 @@ export default {
           if (!this.publicVisualizations) {
             this.setVisualizations();
           }
+          // Request for the queries because we need them to show in visualization
+          if (!this.publicQueries || !this.privateQueries) {
```

Comment:
There's no need anymore to check the privateQueries: they're contained in the public ones

Reply:
Done!